pyatv/mrp/player_state.py

```diff
@@ -105,6 +106,12 @@ def _add_listeners(self):
         self.protocol.add_listener(
             self._handle_set_now_playing_client,
             protobuf.SET_NOW_PLAYING_CLIENT_MESSAGE)
+        self.protocol.add_listener(
```

Comment:
This will not work as it is only sent during connection set up and handled by that code (see comment in `protocol.py`).

Reply:
Not relevant anymore.